src/main/resources/archetype-resources/src/test/java/__module__Test.java

```diff
@@ -0,0 +1,16 @@
+package ${package};
+
+import com.google.j2cl.junit.apt.J2clTestInput;
+import junit.framework.TestCase;
+
+/**
+ * @author Dmitrii Tikhomirov
+ * Created by treblereel 2/11/20
+ */
+@J2clTestInput(${module}Test.class)
+public class ${module}Test extends TestCase {
+
+    public void testGetRanges() {
```

Comment:
consider junit4 test case instead of 3? `@Test`, no `extends TestCase`

Reply:
fixed